Examples/ByFramework/xUnit/FSharp.xUnit/Xunit.FSharp.fsproj

```diff
@@ -16,7 +15,8 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\..\..\TickSpec\TickSpec.fsproj" />
-    <PackageReference Include="Xunit" Version="2.4.0" />
+    <ProjectReference Include="..\..\..\..\TickSpec.XunitWiring\TickSpec.XunitWiring.fsproj" />
+    <PackageReference Include="Xunit" Version="2.4.1" />
```

Comment:
this should be transitive  and hence not explicitly mentioned?

Reply:
Do you mean I should change it to

```
<PackageReference Include="Xunit" Version="2.*" />
```

Reply:
No, explict mention of e.g. 2.2.0 - if you have *, then it will pick the latest, which just means a random TickSpec update triggers cascading. changes in someone's entire test suite

Reply:
changed back to 2.4.0 ... hope this is correct (even though wondering what do I do in my consumer projects, which already have 2.4.1 referenced ...)

Reply:
Ignore the above, according to https://docs.microsoft.com/en-us/nuget/concepts/package-versioning

```xml
<!-- Accepts any version 6.1 and above.
     Will resolve to the smallest acceptable stable version.-->
<PackageReference Include="ExamplePackage" Version="6.1" />

```

Reply:
Consumer projects can happily use a newer version - the point is that if TickSpec.Xunit pulls 2.4.1 and someone is using 2.3.0 in the rest of their tests, then you force them to upgrade (or have inconsistent versions) - if that's avoidable, there's no reason for a test framework to depend on the latest - it should instead depend on the earliest possible.

Then people consuming can update the versions of Xunit they use as an orthogonal concern as and when they see fit

Reply:
Can this be closed - I reverted back to 2.4.0 which was the status quo before this PR

Reply:
👍